client/layout/index.jsx

```diff
@@ -152,6 +154,7 @@ class Layout extends Component {
 					isWooOAuth2Client( this.props.oauth2Client ) &&
 					this.props.wccomFrom,
 				'is-nav-unification': config.isEnabled( 'nav-unification' ),
+				'is-sidebar-collapsed': this.props.sidebarCollapsed,
```

Comment:
Should this sidebarCollapsed business be in the other PR that will deal with collapsing the sidebar?

Reply:
done https://github.com/Automattic/wp-calypso/pull/45989